packages/video_player/video_player_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 6.2.0
+
+* Adds support for picture-in-picture on iOS.
```

Comment:
This should remove "on iOS" since the platform interface isn't platform-specific.

Reply:
@vanlooverenkoen seems like this still needs to change